src/libs/common/include/attributes.h

```diff
@@ -5,6 +5,7 @@
 #include <vector>
 
 #include "storm_assert.h"
+#include "../../util/include/storm/string_compare.hpp"
```

Comment:
should be "storm/string_compare.hpp"
`util` must be required in cmakefile

Reply:
fixed